Accessibility/VirtualContent/explainer.md

```diff
@@ -74,9 +74,15 @@ function begin_load_next_section() {
 
 // Callback triggered when the server responds with next section of content.
 function end_load_next_section(response) {
+  // Signal to the AT that we're loading more content.
```

Comment:
nit: I think "modifying the tree" would be more precise than "loading more content".